java/code/src/com/redhat/rhn/manager/audit/CVEAuditManager.java

```diff
@@ -954,6 +954,7 @@ private static List<CVEAuditSystemBuilder> listSystemsByPatchStatus(List<CVEPatc
             // Group results for the system further by package names, filtering out 'not-affected' entries
             Map<String, List<CVEPatchStatus>> resultsByPackage =
                     systemResults.stream().filter(r -> r.getErrataId().isPresent())
+                            .filter(r -> r.getChannelRank().get() < PREDECESSOR_PRODUCT_RANK_BOUNDARY)
```

Comment:
```suggestion
                            .filter(r -> r.getChannelRank().orElse(0L) < PREDECESSOR_PRODUCT_RANK_BOUNDARY)
```

If for whatever "rank" is empty, we assume 0 and include it into the result check. I think this is better
then risk an exception.

Reply:
Yes that makes sense. I'll make the change